glib/src/auto/functions.rs

```diff
@@ -463,6 +463,12 @@ pub fn main_depth() -> i32 {
     unsafe { ffi::g_main_depth() }
 }
 
+#[doc(alias = "g_markup_escape_text")]
+pub fn markup_escape_text(text: &str) -> crate::GString {
+    let length = text.len() as isize;
+    unsafe { from_glib_full(ffi::g_markup_escape_text(text.to_glib_none().0, length)) }
```

Comment:
How inconvenient that this needs a copy of the string... but that will be fixed properly for the next release together with my other plans :)